### ISSUE TITLE
[Feature Request] Add an option to just output data

### ISSUE THREAD

**Issue by Variiuz** [discord/gamesdk-and-dispatch#50]
An option like --only-data that justs output the data like build id and stuff.
Currently, it outputs stuff formated:

![grafik](https://user-images.githubusercontent.com/36776298/79119270-41beb000-7d90-11ea-80db-3d6040d4396e.png)

It would be nice to just have dispatch outputting data and not fancy formatting

**Comment by lookapanda** [comment]
Agreed, that could be helpful, although that gives the question of what format the output should have?

Side note: With a simple regex you can get most of the data fairly easy
